src/Microsoft.Framework.BufferEntryCollection.Internal/BufferEntryCollection.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Framework.Internal
+{
+    /// <summary>
+    /// An ordered collection of strings with an enumerator that iterates over it.
+    /// </summary>
+    internal class BufferEntryCollection : IEnumerable<string>
```

Comment:
To my knowledge, we put public class in `Internal` namespace so that the class can be consumed but we don't guarantee the API's stable. Now that an `internal` class is placed here doesn't it defeat the purpose of `Internal` namespace? 
Or do we can different guidelines for how to use `Internal` namespace?